packages/fxa-settings/src/components/DropDownAvatarMenu/index.test.tsx

```diff
@@ -89,12 +88,13 @@ describe('DropDownAvatarMenu', () => {
         <DropDownAvatarMenu />
       </AppContext.Provider>
     );
-    const dropDown = screen.queryByTestId('drop-down-avatar-menu');
 
     fireEvent.click(screen.getByTestId('drop-down-avatar-menu-toggle'));
-    expect(dropDown).toBeInTheDocument;
+    expect(screen.queryByTestId('drop-down-avatar-menu')).toBeInTheDocument();
```

Comment:
Per L54, `const dropDownId = 'drop-down-avatar-menu';`
```suggestion
    expect(screen.queryByTestId(dropDownId)).toBeInTheDocument();
```